### DISCUSSION TITLE
Align iOS "Transmit Ready" sound with Qt client

### DISCUSSION THREAD

**Discussion by bear101** [BearWare/TeamTalk5#1081] [Ideas]
@Oreonan Some users are complaining that the guitar sound is too loud in the Qt client and would like to change it to the sound from iOS.
Basically change [txqueue_start.wav](https://github.com/BearWare/TeamTalk5/blob/master/Setup/Client/Sounds/txqueue_start.wav) with [txqueue_start.mp3](https://github.com/BearWare/TeamTalk5/blob/master/Client/iTeamTalk/iTeamTalk/txqueue_start.mp3)

Sounds ok to you?

**Comment by CoBC** [comment]
Yes, it's OK for me, less loud and more short.

**Comment by bear101** [comment]
ok done 7a9de61f1
